src/LineBot/Messages/Template/Carousel/CarouselColumn.cs

```diff
@@ -27,6 +27,8 @@ public sealed class CarouselColumn
         private Uri _thumbnailUrl;
         private string _title;
         private string _text;
+        private string _color = "#FFFFFF";
```

Comment:
I wonder if we should change the code to remove `null` values when serializing the object. We don't need this default then. Could you take a look at that?

Reply:
Agree, I will check these attribute.